docs/DeveloperGuide.md

```diff
@@ -300,38 +300,34 @@ It will also include settings to create a custom look for the application.
+| `* *`    | student living in dorm with many contacts | filter contacts by dorm room number                                  | I can find where are my dorm mates                                                              |
+| `* *`    | student living in dorm with many contacts | search a contact by birthday                                         | I know whose birthday is in which month                                                         |
+| `* *`    | student living in dorm                    | add a profile picture for each contact                               | I can recognise and identify the contact person                                                 |
+| `* *`    | student living in dorm                    | add telegram handle for each contact                                 | I can contact them on telegram as an alternative to their phone number                          |
+| `*`      | student living in dorm                    | sync room updated room numbers across multiple users (decentralised) | I can just update my room number and not worry about other user’s room numbers                  |
+| `*`      | student living in dorm who enjoys events  | add upcoming events                                                  | I can remember what events are coming up                                                        |
+| `*`      | student living in dorm who enjoys events  | view upcoming events                                                 | I can plan for them accordingly                                                                 |
+| `*`      | student living in dorm who enjoys events  | view upcoming birthdays                                              | I can plan a celebration and wish them happy birthday                                           |
+| `*`      | new user of this application              | type help to get a list of all commands and how to use them          | I can find the command I want to use                                                            |
+| `*`      | user exploring this application           | toggle between dark and light mode                                   | I can make my view of the application more comfortable to the eye                               |
+| `*`      | user exploring this application           | apply custom background                                              | I can personalise the application to my liking                                                  |
+| `*`      | expert user of this application           | export contact details as a .csv file                                | I can save and share the contacts in a backup location                                          |
+| `*`      | expert user of this application           | import contact details from a .csv file                              | I can duplicate contacts into another copy of the application on another device                 |
```

Comment:
Will "export as csv" or "import as csv" be too UI-specific? I remember there is a part of requirement specification that mentions UI-specific details shouldn't be included, not sure whether it is referring to user stories.

Reply:
updated